src/RemoteTech/FlightComputer/PIDController.cs

```diff
@@ -0,0 +1,85 @@
+﻿/// <summary>
+/// Proportional controller for KSP autopilots
+/// </summary>
+
+using System;
+
+namespace RemoteTech.FlightComputer
+{
+    // -----------------------------------------------
+    // Copied from MechJeb master on 18.04.2016
+    public class PIDControllerV3 : IConfigNode 
```

Comment:
I'd prefer if file name and class name were the same. Either `PIDController` or `PIDControllerV3` is fine by me, just make them the same.